tidypolars/tibble.py

```diff
@@ -949,6 +960,110 @@ def write_parquet(self,
         """Write a data frame to a parquet"""
         return super().write_parquet(file, compression = compression, use_pyarrow = use_pyarrow, **kwargs)
 
+    def group_by(self, group, *args, **kwargs):
```

Comment:
`*args` and `**kwargs` need be passed to the `TibbleGroupBy()` call

Reply:
It is already there in the __init__